.circleci/config.yml

```diff
@@ -16,21 +16,41 @@ parameters:
   postgres-version:
     type: string
     default: "15.5"
+  localstack-version:
+    type: string
+    default: "3.6.0"
+  java-options:
+    type: string
+    default: -Xmx512m -XX:ParallelGCThreads=2 -XX:ConcGCThreads=2 -Djava.util.concurrent.ForkJoinPool.common.parallelism=2 -Dorg.gradle.daemon=false -Dkotlin.compiler.execution.strategy=in-process -Dorg.gradle.workers.max=1
 
-jobs:
-  validate:
-    executor:
-      name: hmpps/java_postgres
-      jdk_tag: << pipeline.parameters.java-version >>
-      postgres_tag: << pipeline.parameters.postgres-version >>
-      postgres_db: "education_and_work_plan_api_db"
-      postgres_username: "admin"
-      postgres_password: "admin_password"
+executors:
```

Comment:
I couldnt find an hmpps orb that is java and postgres and localstack (there is one for java and postgres; and one for localstack; but not one for everything); and I couldn't workout how to configure the job with multiple orbs.
So I've gone with this approach

Reply:
One should exist, see [here](https://github.com/ministryofjustice/hmpps-book-a-video-link-api/blob/main/.circleci/config.yml#L20) for an example.

Reply:
@matt-crowson-moj - Amazing; thank you; that looks to be just the ticket 👍 
I'll change it to use that from your example. Thanks